rest-service/manager_rest/rest/resources_v3/cluster.py

```diff
@@ -20,9 +20,9 @@
 from ..responses_v3 import BaseResponse
 
 try:
-    from cloudify_premium import (ClusterResourceBase,
-                                  ClusterState,
-                                  ClusterNode)
+    from cloudify_premium.ha.web import (ClusterResourceBase,
```

Comment:
eh maybe just remove the whole file

Reply:
I don't want to remove the old cluster together with this merge.
I have a separate task for it